manifests/c/chmln/sd/0.7.5/chmln.sd.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: chmln.sd
+PackageVersion: 0.7.5
+PackageLocale: en-US
+Publisher: Gregory
+PublisherUrl: https://github.com/chmln
+PublisherSupportUrl: https://github.com/chmln/sd/issues
+Author: Gregory
+PackageName: Search & Displace

```

```suggestion
PackageName: sd - search & displace
```

We can follow the name that the repo README uses: https://github.com/chmln/sd#sd---search--displace. This may also help users doing `winget list sd`  (Though in my testing, the package shows up in the output with the current PackageName, but I'll still prefer the README naming used by the author)